openwisp_controller/config/tests/test_selenium.py

```diff
@@ -83,16 +83,15 @@ def test_create_new_device(self):
         self.web_driver.find_element(
             by=By.XPATH, value='//*[@id="config-group"]/fieldset/div[2]/a'
         ).click()
-
         try:
             WebDriverWait(self.web_driver, 2).until(
-                EC.presence_of_element_located(
-                    (By.XPATH, f'//*[@value="{default_template.id}"]')
+                EC.text_to_be_present_in_element_value(
+                    (By.CSS_SELECTOR, 'input[name="config-0-templates"]'),
+                    f'{required_template.id},{default_template.id}',
```

Comment:
I don't understand 100% what is going on here, can you please clarify:

- add a human friendly comment that clarifies this assertion is expecting
- why `presence_of_element_located` was giving inconsistent results
- why you think `text_to_be_present_in_element_value` would work better
- if I understand correctly, we expect the templates currently assigned to the device to be always the same, having the required template listed first and the default template after, is my understanding correct? If so, please also add a comment to summarize this concept as this part of the code is not easy to grasp just by reading these lines